src/platform/utilities/ui/Breadcrumbs.jsx

```diff
@@ -0,0 +1,118 @@
+// TODO: Move to Jean Pants when approved
+import PropTypes from 'prop-types';
+import React from 'react';
+import _debounce from './debounce';
+import uniqid from 'uniqid';
+
+class Breadcrumbs extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      mobileShow: false,
+    };
+
+    this.id = this.props.id || uniqid('va-breadcrumbs-');
```

Comment:
This is a little risky in that props could change and the instance variables wouldn't change to reflect that. You could make these `const` variables inside the render method (or other methods where they're needed), I think.